api/src/main/scala/hmda/api/http/institutions/SubmissionPaths.scala

```diff
@@ -141,4 +141,39 @@ trait SubmissionPaths
       }
 
     }
+
+  def submissionSingleEditPath(institutionId: String) =
+    path("filings" / Segment / "submissions" / IntNumber / "edits" / Segment) { (period, submissionId, editType) =>
+      val path = s"institutions/$institutionId/filings/$period/submissions/$submissionId/edits/$editType"
+      extractExecutionContext { executor =>
+        implicit val ec: ExecutionContext = executor
```

Comment:
The point isn't really to reduce line count (though that is often a nice benefit of sharing code), the goal is to abstract out ideas (to reduce code complexity) and reduce duplication (to make it more maintainable).

The next several lines (the entire for comprehension) also seem to be shared with the previous method, so they could be included in the extracted code as well.